hpx/compute/host/block_allocator.hpp

```diff
@@ -15,11 +15,19 @@
 #include <hpx/parallel/algorithms/for_each.hpp>
```

Comment:
It's still needed: https://github.com/STEllAR-GROUP/hpx/blob/fixing_block_allocator/hpx/compute/host/block_allocator.hpp#L226